airflow-core/src/airflow/cli/cli_config.py

```diff
@@ -177,11 +177,6 @@ def string_lower_type(val):
     type=str,
     default="[CWD]" if BUILD_DOCS else os.getcwd(),
 )
-ARG_DRY_RUN = Arg(
-    ("-n", "--dry-run"),
-    help="Perform a dry run for each task. Only renders Template Fields for each task, nothing else",
-    action="store_true",
-)
```

Comment:
It was a useless command since we alerady have a separate command `airflow tasks render`